sss/src/se05x/fsl_sss_se05x_apis.c

```diff
@@ -4906,14 +4910,38 @@ sss_status_t sss_se05x_asymmetric_sign_digest(
     case kSSS_CipherType_EC_BRAINPOOL:
 #endif
     {
+#if defined(CFG_OPTEE_REVISION_MAJOR) && defined(CFG_OPTEE_REVISION_MINOR)
         SE05x_ECSignatureAlgo_t ecSignAlgo = se05x_get_ec_sign_hash_mode(context->algorithm);
-        status                             = Se05x_API_ECDSASign(&context->session->s_ctx,
+	size_t len = *signatureLen;
+	size_t sz = len;
+	size_t retry = 100;

```

Comment:
100?

Reply:
it usually takes between 3 and 20 retries (worst case I have measured)...we could also loop for ever.

there are probability analysis done ie https://transactionfee.info/charts/bitcoin-script-ecdsa-length so in theory we shouldnt need to retry for too long...

Reply:
Cool, so please add a comment saying why you decided to go with 100 for posterity :-)

Reply:
I dont really like this workaround to be honest - discarding valid signatures because we dont handle them properly in OP-TEE is not great. Let me see if I can at least refine it a bit more.

Reply:
The issue with the ones that require padding is that the only way to verify the signatures would be to regenerate the public key from the signature and then compare to decide if the null at the end of the r,s fields is a pad or a valid value.
it is doable but I dont think is worth the effort.

Anyway, I am going to update the PR to retry only on the pad condition which is good enough.